.github/workflows/ci.yml

```diff
@@ -247,13 +247,13 @@ jobs:
         uses: ./.github/actions/run-script
         if: github.event_name != 'pull_request' && github.ref == 'refs/heads/main'
         with:
-          script-name: "test:visual:ci --filter='{./packages/components/*}[HEAD^1]'"
+          script-name: "test:visual:ci --filter='{./packages/components/*}[HEAD^1]' --filter='{./packages/tools/*}[HEAD^1]'"
```

Comment:
There is no changelog nor description I can see of what this is doing - please could you explain it?

Reply:
This gets the `icons-webc` tests to run in CI. Neither component nor visual have been running in CI for 9 months 🙃

Reply:
oof, ty!